classes/class06/README.md

```diff
@@ -1 +1,266 @@
-#Class 5
+# Class5 - CSS, Development Style and Grace
+
+## Separation of Concerns
+
+Client-side web development is dominated by three technologies.
+
+- **HTML** specifies the _structure_ and _content_ of a webpage.
+- **CSS** specifies the _presentation_ or _appearance_ of elements on a webpage.
+- **JavaScript** lets the developer create _interactive_ elements. It's how the client communicates with the server and the user.
+
+Before CSS, HTML was also used to control the appearance of webpages. This is why websites from the '90s look terrible.
+
+These three technologies embody the principle of _separation of concerns_ — they are each completely responsible for a single aspect of a complex web app.
+
+## App file structure/hierarchy
+
+- **/models** - contains ORM models (mongoose `Schemas`)
+- **/views** - view-templates
+- **/public** - all static content
+  - **/images**
+  - **/stylesheets**
+  - **/javascripts**
+- **/routes** - all express routes, separated by app module/area
```

Comment:
Are we going to briefly touch on MVC? If so, clarify that our controllers live in routes. They can be separated, but for simple applications, it's acceptable to write controller logic directly in to route assignment. Mostly relevant to using the breakout router.